requirements-build.in

```diff
@@ -1,9 +1,10 @@
-# Generated by pip_find_builddeps.py on Oct 20 2021 09:32:35
+# Generated by pip_find_builddeps.py on Oct 26 2021 19:48:13
 Cython
 cffi>=1.12
 flit_core<4,>=3.2.0
 packaging>=20.0
 poetry-core
+pycairo
```

Comment:
`pycairo` is a runtime dep of  `PyGObject`, not a builddep

https://gitlab.gnome.org/GNOME/pygobject/-/blob/master/setup.py (it's in install_requires, not setup_requires)

Reply:
hmm, I see what you're saying and I agree we should move it. But it was the result of the builddep.py script so if we move it we'll have to always move it after running that script.

Reply:
Did you run the builddeps script before or after pip-compiling the requirements.in? If any runtime dependencies are missing in requirements.txt, that script will mis-identify them as build dependencies

Reply:
tried your suggestion `pycairo` is in the correct place now :D

Reply:
Why is it still in the build requirements file? Did the find_builddeps script put it there even when it was already in requirements.txt?

Reply:
it was added in build requirements even though it was in requirements.txt I had to remove it manually this time.

Reply:
Oh, my bad, it is both a runtime and a build time dependency (the build dependency is defined here https://gitlab.gnome.org/GNOME/pygobject/-/blob/master/pyproject.toml#L19). Please leave it in the build requirements, sorry :sweat_smile:

Reply:
ok, I've added it back to build requirements as well.